lib/r10k/tarball.rb

```diff
@@ -70,28 +87,54 @@ def cache_basename
     end
 
     # Extract the cached tarball to the target directory.
-    #
+    # 
     # @param target_dir [String] Where to unpack the tarball
+    # @param remove_wrapper_dir [Boolean] Whether or not to remove the wrapper
```

Comment:
does this need to be added to the parameter list for `unpack`?

Reply:
Ah; nope, that's an rdoc @param leftover from a previous implementation. `remove_wrapper_dir` is now an attr reader method. I will remove the @param note.